### ISSUE TITLE
Problem patching simple tables

### ISSUE THREAD

**Issue by gonzaloserrano** [paulfitz/daff-php#15]
Hi,

I have the following code: https://gist.github.com/gonzaloserrano/07948738bd3abc91545b

I'm trying to patch this table:
```
$remoteTable = [
    ['col1', 'col2', 'col3', 'col4', 'col5', 'col6'],
    [0, 0, 0, 0, 2, 0]
];
```
into this one:
```
$localTable = [
    ['col1', 'col2', 'col3', 'col4', 'col5', 'col6'],
    [0, 0, 0, 0, 1, 0]
];
```
And i have errors similar to:
```
PHP Notice:  Undefined offset: 3 in /Users/gonzalo/dev/sp/config-manager-backend/vendor/paulfitz/daff-php/gen/lib/coopy/PhpTableView.class.php on line 131

Notice: Undefined offset: 3 in /Users/gonzalo/dev/sp/config-manager-backend/vendor/paulfitz/daff-php/gen/lib/coopy/PhpTableView.class.php on line 131
PHP Notice:  Undefined offset: 3 in /Users/gonzalo/dev/sp/config-manager-backend/vendor/paulfitz/daff-php/gen/lib/coopy/PhpTableView.class.php on line 132

Notice: Undefined offset: 3 in /Users/gonzalo/dev/sp/config-manager-backend/vendor/paulfitz/daff-php/gen/lib/coopy/PhpTableView.class.php on line 132
PHP Notice:  Undefined offset: 3 in /Users/gonzalo/dev/sp/config-manager-backend/vendor/paulfitz/daff-php/gen/lib/coopy/PhpTableView.class.php on line 133
```

If you remove ANY column from the tables except the 5th (which is the one which has the change) it works flawlessly, but like its found in the gist it gives those errors.

I'm using last master version which has the 1.2.4 commit message on it.

Thanks!

**Comment by paulfitz** [comment]
Thanks for reporting this @gonzaloserrano. Replicated and working on a fix.

**Comment by paulfitz** [comment]
Should be fixed in 1.2.5, please reopen if not.

**Comment by gonzaloserrano** [comment]
The test i did now works file, will report if i find any further problems. Thanks!
